includes/yesno.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Yes / No Functions for use in WC option filters.
+ *
+ * @since 1.8.1
+ * @package WC_Calypso_Bridge
+ */
+
+/**
+ * Always return 'yes'.
+ *
+ * @return string yes
+ */
+function wc_calypso_bridge_return_yes() {
```

Comment:
May I know what these are for?

Reply:
Filters like this https://github.com/Automattic/wc-calypso-bridge/pull/729/files#diff-64414c498da63dbd91258d202730af80ce94415aafd9a2217970343ff62201c3R12 there will be a few of them and I'd rather just comment the reason for adding the filter than having a named function for each thing we disable. 

The required disable logic is also reversed quite often which means you end up with functions that look like typos:

```
function disable_feature_a() {
    return 'no';
}
function disable_feature_b() {
    return 'yes';
}
```